javatests/arcs/core/entity/BUILD

```diff
@@ -43,6 +43,7 @@ arcs_kt_jvm_library(
         ["*.kt"],
         exclude = TEST_SRCS,
     ),
+    visibility = ["//javatests/arcs/android/entity:__pkg__"],
```

Comment:
@csilvestrini Is this legit?

Reply:
This is legit, but copybara might not like it. Let's wait and see...